ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDF.java

```diff
@@ -434,10 +434,19 @@ protected void obtainTimestampConverter(ObjectInspector[] arguments, int i,
     case TIMESTAMP:
     case DATE:
     case TIMESTAMPLOCALTZ:
+    case INT:
```

Comment:
Do these newly added datatypes for timestamp conversion have test associated with them? if not, can you add them?

Reply:
There data are already implemented in https://github.com/apache/hive/blob/7b3ecf617a6d46f48a3b6f77e0339fd4ad95a420/serde/src/java/org/apache/hadoop/hive/serde2/objectinspector/primitive/PrimitiveObjectInspectorUtils.java#L1177. I am just adding validation check to avoid runtime failures. So no test is required in this case.